### ISSUE TITLE
One-hot format for output data

### ISSUE THREAD

**Issue by emanjavacas** [hipster-philology/pandora#14]
It seems that keras needs one-hot format for the targets in order to compute the loss,
Pytorch however requires integer format. It makes sense to default to the pytorch format
and have the keras implementation call to_categorical on it rather than having pytorch
undoing the categorical (which means twice as much computation)

**Comment by mikekestemont** [comment]
I agree (also needs much less memory). Feel free to make the necessary
changes in the pytorch branch.

Prof. Dr. Mike Kestemont | www.mike-kestemont.org | Twitter:
@Mike_Kestemont | mike.kestemont@uantwerp.be | mike.kestemont@gmail.com |
University of Antwerp | City Campus, Prinsstraat 13, room D. 118 I B-2000
Antwerp, Belgium | tel. +32 (0)3 265.42.54

> Check out our documentary on Digital Humanities and Hildegard of Bingen:
watch it in HD on Vimeo: https://vimeo.com/70881172

**Comment by emanjavacas** [comment]
Done as per 007db3f1ffd82dfe5690a3ccd5afa7cefb22da41. To keep it simple I just decided to add an argument to Preprocessor constructor `categorical` specifying whether the data should be turned into binary format or not.
